src/module_3/module_3_milestone2.py

```diff
@@ -0,0 +1,104 @@
+import pandas as pd
+import numpy as np
+from sklearn import linear_model
+from sklearn.pipeline import make_pipeline
+from sklearn.preprocessing import StandardScaler
+from sklearn import metrics
+import joblib
+
+
+DATA_PATH = "/home/unai/datasets/feature_frame.csv"
+training_columns = [
+    "abandoned_before",
+    "ordered_before",
+    "global_popularity",
+    "user_order_seq",
+    "normalised_price",
+]
+label_col = "outcome"
+model_filename = "groceries_model.pkl"
+
+
+def main():
+    data = load_data(DATA_PATH)
+    # processed_data = preprocessing(data)
+    # model = training_model(processed_data)
+    # save_model(model, model_filename)
+    model = load_model(model_filename)
+    threshold = threshold_calculation(data, model)
+    # with these 3 variables we can predict new values with the model_prediction function
```

Comment:
I do not understand what are you doing here. You should split this in two files one for training and another for inference

Reply:
In the inference process what should I do?